packages/frontend-2/lib/workspaces/graphql/mutations.ts

```diff
@@ -0,0 +1,12 @@
+import { graphql } from '~~/lib/common/generated/gql'
+
+export const workspaceUpdateRoleMutation = graphql(`
+  mutation WorkspaceUpdateRole($input: WorkspaceRoleUpdateInput!) {
```

Comment:
This seems to be a duplicate mutation of the settings one

Reply:
Forgot to delete the other one 😅